genume/registry/value.py

```diff
@@ -1,22 +1,11 @@
-from genume.registry.base import BaseEntry
+from genume.registry.base import BaseEntry, InfoLevel
 
 
 class ValueEntry(BaseEntry):
-    """A type of entry which contains a single string."""
-
-    def __init__(self, parent=None, value=""):
-        super().__init__(parent)
-        self.value = value
-
-    def __repr__(self):
-        return self.value
-
-
-class ListEntry(BaseEntry):
```

Comment:
You merged Value and List entries into one, is there a reason behind this?

Reply:
The protocol does not distinguish between them anymore.

Reply:
This will lead to all values being arrays and mostly arrays of one element. Then the supported formats, like json, will not have a very nice structure.

I would suggest at some point, later, support both the VALUE and VALUES command. The only change to the impl would a boolean in the value entry that would suggest if this a multiple or single value. And then based on that all exporters could could just check the boolean.

I could implement this if you would like in my next pr, where I want to add more exporters.

Reply:
The protocol should stay as simple as possible. I think that having two different commands doing very similar things is useless and confusing.
What I think is some auto converversion at either registry or export level.